tests/ts/fincore/count

```diff
@@ -10,8 +10,8 @@ ts_check_test_command "$TS_CMD_FINDMNT"
 ts_check_test_command "$TS_HELPER_SYSINFO"
 
 FS="$("$TS_CMD_FINDMNT" -nr -o FSTYPE -T "$PWD")"
-if [[ "$FS" = "tmpfs" ]]; then
-	ts_skip "fincore does not work on tmpfs"
+if [[ "$FILE_FS" = "tmpfs" || "$FILE_FS" = "overlay" || "$FILE_FS" = "" ]]; then
```

Comment:
It should be `$FS` instead of `$FILE_FS` here.
(Or even better, they should be aligned)